alerting/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12
+FROM node:16
```

Comment:
Did you tried node:20 ? ^^

Reply:
I have not, but we upgraded the rest of the app is using node:16 so I upgraded the alerting module to be consistent with the rest